src/penn_chime/model/parameters.py

```diff
@@ -354,11 +354,11 @@ def create(
         )
         icu = Disposition.create(
             days=a.icu_days,
-            rate=a.icu_rate,
```

Comment:
This is a fundamental change to the meaning of the model parameters. Users with original configuration files will get different results.